index.js

```diff
@@ -0,0 +1,39 @@
+import { createStore } from '../node_modules/redux/es/redux.mjs';
+import { getNextState } from './redux/reducer.js';
+import { startEditItem, endEditItem } from './redux/actions.js';
+
+const store = createStore(getNextState);
+
+store.subscribe(() => {
+  render();
+})
+
+const root = document.querySelector('#root');
+
+render();
+
+function render() {
```

Comment:
To make render function more universal and reusable, will be better to pass container as argument